src/components/Header/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Header = ({heading, color, size}) => {
```

Comment:
@kerry-okpere, I noticed that even though you have the `color`, you are not using it anywhere in your component. Do you plan to do this in a separate PR?